CalibTracker/Records/interface/SiPixelMappingSoARecord.h

```diff
@@ -0,0 +1,9 @@
+#ifndef CalibTracker_Records_interface_SiPixelMappingSoARecord_h
+#define CalibTracker_Records_interface_SiPixelMappingSoARecord_h
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+
+class SiPixelMappingSoARecord : public edm::eventsetup::EventSetupRecordImplementation<SiPixelMappingSoARecord> {};
```

Comment:
I see `SiPixelRawToCluster` consumes `SiPixelMappingDevice` in this record, but I don't see any ESProducer producing anything into `SiPixelMappingSoARecord`. I see `SiPixelCablingSoAESProducer` produces `SiPixelMappingHost` into `CkfComponentsRecord` that then gets automatically transferred into `SiPixelMappingDevice`. But which is the intention
* Place the product in `CkfComponentsRecord`, in which case this Record and the consumer needs to be updated
* Place the product in `SiPixelMappingSoARecord`, in which case this Record needs to be made to depend on some other Records, and the producer needs to be updated

?